distro/adaptation/fedora

```diff
@@ -55,6 +55,7 @@ libc-bin/unstable: glibc-common
 libc-dev-bin: glibc-common
 libc-dev:i386: glibc-devel.i686
 libcunit1-dev: CUnit-devel
+libconfig-dev: libconfig-devel
 libdbd-mysql-perl: libdbi-dbd-mysql
 libdbi-perl: perl-DBI
 libdbus-1-dev: dbus-devel
```

Comment:
thanks, it looks good, for the subject, looks there's a typo defora, should it be fedora?

Reply:
> thanks, it looks good, for the subject, looks there's a typo defora, should it be fedora?
Thanks for the review.
I am sorry for the typo. I will fix it and resend it.